.github/workflows/apidocstest.yml

```diff
@@ -0,0 +1,90 @@
+name: api docs
+on:
+  pull_request:
```

Comment:
This workflow won't run on commits pushed to master, only PRs branched from it - is that expected? Just making sure here since it's been overlooked before.

Reply:
Since this workflow runs tests, I think you would want to also add `on.push`.